IFC4x3/Properties/a/AssemblyReference_0FbOAb3P1Cbhju1SFIPkJn/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AssemblyReference_0FbOAb3P1Cbhju1SFIPkJn" Name="AssemblyReference" UniqueId="0f9582a5-0d90-4c96-bb78-05c3d266e4f1" PropertyType="p_referencevalue" PrimaryDataType="IfcExternalReference" />
```

Comment:
Same comment as Assemble Reference, no primary measure type and could it be IfcDocumentReference and not a property?

Reply:
Reused of existing AssemblyInstruction which is "IfcExternalReference. Commit : 0bb5ae0d772bc5a99c0bea0886d3f8ed41fd2f5d